docs/reference/copy-guide.md

```diff
@@ -0,0 +1,275 @@
+# Gutencopy Guidelines
+
+## Longer Text
+Guidelines for writing multi-line/step instructions or narrative introductions/orientation to pages or features.
+
+This will obviously vary quite a lot depending on the context, but here are some general tips:
+
+#### ONE: Contractions are your friends!
```

Comment:
If these are to be numbered, can we use numbers? eg `### 1. Contractions`. That said I'm not sure why you'd number them if the list doesn't appear elsewhere. I'd expect the list first (like a table of contents), then each point expanded on after.

Also shouldn't these be H3s not H4s?